src/contracts/middleware/VoteWeigherBase.sol

```diff
@@ -191,7 +187,8 @@ contract VoteWeigherBase is VoteWeigherBaseStorage {
         uint8 quorumNumber,
         StrategyAndWeightingMultiplier[] memory _newStrategiesConsideredAndMultipliers
     ) internal {
-        require(quorumNumber <= quorumCount, "VoteWeigherBase._addStrategiesConsideredAndMultipliers: quorumNumber exceeds quorumCount");
+        require(quorumNumber < quorumCount, "VoteWeigherBase._addStrategiesConsideredAndMultipliers: quorumNumber exceeds quorumCount");
```

Comment:
maybe make this a modifier? I see it used a couple places

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/53/commits/d4be2b7f8585ad8ef9171fba9171db253b572e0b